test/utility/misc.c

```diff
@@ -443,6 +443,22 @@ testutil_copy_if_exists(WT_SESSION *session, const char *name)
         testutil_copy_file(session, name);
 }
 
+/*
+ * testutil_verify_model --
+ *     Run the model verification tool on the database. The database must be closed, and while it
```

Comment:
Grammar nitpick in the comment - "and while it" -> "and it" (unless I've misunderstood the sentence, in which case - sorry!)